tracer/test/snapshots/Security.AspNetWebApi.Classic.enableSecurity=True.__test=server.request.path_params_url=_api_route_2-arg&[$slice]_body=null.verified.txt

```diff
@@ -0,0 +1,267 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet-webapi.request,
+    Resource: GET /api/route/{id},
+    Service: sample,
+    Type: web,
+    ParentId: Id_3,
+    Tags: {
+      aspnet.controller: route,
+      aspnet.route: api/{controller}/{id},
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 200,
+      http.url: http://localhost/api/route/2?arg&[$slice],
```

Comment:
shouldn't this test trigger a security event? 🤔

Reply:
Good point, I thought I'd checked that it did, but obviously not :(

Reply:
so turns out `arg&[$slice]` doesn't generate an event as the attack is in a key. I think we added it as a tests because empty keys where breaking our integration. I've updated the tests to use `arg=[$slice]` instead.